model/job-management/src/main/proto/beam_artifact_api.proto

```diff
@@ -147,3 +154,9 @@ message CommitManifestResponse {
   // ArtifactRetrievalService.
   string retrieval_token = 1;
 }
+
+// repeated from beam_runner_api.proto - is there a way to share the extension?
```

Comment:
Looks like you can do it if you include the package name: https://stackoverflow.com/questions/17126230/extending-protobuf-fieldoptions-in-imported-proto-file

Reply:
Following doesn't make the extension available, even though the import is OK: `import "beam_runner_api.proto"`

Reply:
What error are you getting?

It looks like the import needs to be fully qualified, i.e. `org.apache.beam.model.pipeline.v1.beam_constant` 
https://github.com/apache/beam/blob/2b441c25c6353f1f293474c144046e241a0f2a02/model/pipeline/src/main/proto/metrics.proto#L221

Reply:
That was it, thanks!